### PR TITLE
Partials 100%

### PR DESCRIPTION
**Pull request by Pwntus**

### CODE REVIEW

File: themes/vue/layout/partials/ecosystem_dropdown.ejs [hunk 1]
```diff
@@ -23,7 +23,7 @@
     </ul></li>
     <li><h4>Nyheter</h4></li>
     <li><ul>
-      <li><a href="https://news.vuejs.org" class="nav-link" target="_blank">Veckovis nyheter</a></li>
+      <li><a href="https://news.vuejs.org" class="nav-link" target="_blank">Veckovisa Nyheter</a></li>
```
Comment (sustained) [inline themes/vue/layout/partials/ecosystem_dropdown.ejs]: Regarding this whole "title case" thing...

I was always taught that in Swedish we do NOT use title case, so titles should always be in lowercase.

So then where in English we'd write "List of Interesting Things", in Swedish we would instead write "Lista över intressanta saker".

At least I was pretty sure that's how it was. I've asked some Swedish friends to confirm/deny that for me.

File: themes/vue/layout/partials/learn_dropdown.ejs [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <li class="nav-dropdown-container learn">
-  <a class="nav-link<%- page.path.match(/\/(guide(?!\/team.html$)|api|style-guide|examples|cookbook)(\/|$)/) ? ' current' : '' %>">Lära sig</a><span class="arrow"></span>
+  <a class="nav-link<%- page.path.match(/\/(guide(?!\/team.html$)|api|style-guide|examples|cookbook)(\/|$)/) ? ' current' : '' %>">Lär</a><span class="arrow"></span>
```
Comment (sustained) [inline themes/vue/layout/partials/learn_dropdown.ejs]: att lära - to teach
att lära sig - to learn

So should it not be "lär sig"? I did forget to choose the imperative tense though.

File: themes/vue/layout/partials/sidebar.ejs [hunk 3]
```diff
@@ -9,10 +9,10 @@
         <h2>
           <% titles = {
             api: 'API',
-            examples: 'Examples',
+            examples: 'Eksempel',
```
Comment (sustained) [inline themes/vue/layout/partials/sidebar.ejs]: Heh, are you Danish/Norwegian? It's `(ett) exempel` in Swedish. :smile:

File: themes/vue/layout/partials/ecosystem_dropdown.ejs (hunk 1 above)
Comment (sustained) [reply themes/vue/layout/partials/ecosystem_dropdown.ejs]: Confirmed - title case is not used in Swedish; titles should be all lowercase.

File: themes/vue/layout/partials/learn_dropdown.ejs (hunk 2 above)
Comment (Pwntus) [reply themes/vue/layout/partials/learn_dropdown.ejs]: I don't think "lära sig" by itself sound correct.
`att lära sig` means more "the action to learn" and `lära sig` would still mean "to learn".
`lär sig` would be tilted towards meaning "someone who is learning".
I think just "lär" or "lär dig" (= more like "teach yourself") is best.

File: themes/vue/layout/partials/ecosystem_dropdown.ejs (hunk 1 above)
Comment (Pwntus) [reply themes/vue/layout/partials/ecosystem_dropdown.ejs]: Thanks, will change this.

File: themes/vue/layout/partials/sidebar.ejs (hunk 3 above)
Comment (Pwntus) [reply themes/vue/layout/partials/sidebar.ejs]: I've used Norwegian for a couple of years now, so I'm having some issues adjusting the small things :grin: 
Will change.

File: themes/vue/layout/partials/learn_dropdown.ejs (hunk 2 above)
Comment (sustained) [reply themes/vue/layout/partials/learn_dropdown.ejs]: Sure `lär dig` is probably a better choice than `lär sig` but I was speaking generally thus using infinitives. My point was that there needed to be __some__ subject else the meaning is "teach". :+1:

**Comment by sustained** [comment]
This all looks good for merging to me except for imho `lär` :arrow_right: `lär dig` btw.

**Pull request merged**